pkg/repository/prisma/dbsqlc/queue.sql

```diff
@@ -1,40 +1,52 @@
 -- name: UpsertQueue :exec
+WITH queue_exists AS (
+    SELECT
+        1
+    FROM
+        "Queue"
+    WHERE
+        "tenantId" = @tenantId::uuid
+        AND "name" = @name::text
+), queue_to_update AS (
+    SELECT
+        *
+    FROM
+        "Queue"
+    WHERE
+        EXISTS (
+            SELECT
+                1
+            FROM
+                queue_exists
+        )
+        AND "tenantId" = @tenantId::uuid
+        AND "name" = @name::text
+    FOR UPDATE SKIP LOCKED
```

Comment:
The solve for that was a statement timeout on the heartbeats, in case we accidentally have a connection holding a lock when it shouldn't. Will do the same thing here.